libmscore/rendermidi.cpp

```diff
@@ -953,6 +953,22 @@ int articulationExcursion(Note *noteL, Note *noteR, int deltastep)
       }
 
 //---------------------------------------------------------
+// totalTiedNoteTicks
+//      return the total of the actualTicks of the given note plus
+//      the chain of zero or more notes tied to it to the right.
+//---------------------------------------------------------
+int totalTiedNoteTicks(Note* note)
+      {
+      int total = note->chord()->actualTicks();
+      while ( note->tieFor()) {
```

Comment:
That will fail in the case of a note 1 tied to note 2 tied to note 1. Also remove the space in while (note